yarn.lock

```diff
@@ -7341,7 +7341,7 @@ longest@^1.0.1:
   resolved "https://registry.yarnpkg.com/longest/-/longest-1.0.1.tgz#30a0b2da38f73770e8294a0d22e6625ed77d0097"
```

Comment:
Please remove this lock, since it is affected by `maleo-redux-plugin` which already removed